app/models/book.rb

```diff
@@ -1,4 +1,12 @@
 class Book < ApplicationRecord
+  has_many :reviews
```

Comment:
Better add `dependent: :destroy` here to make it cascading remove `reviews` when the book record is deleted.
```suggestion
  has_many :reviews, dependent: :destroy
```